src/store/getters/state-getters.ts

```diff
@@ -433,7 +433,7 @@ export const isDissolutionValid = (state: StateIF): boolean => {
   const isCourtOrderValid = isStaff ? getCourtOrderStep(state).valid : true
   const isStaffPaymentValid = isStaff ? getStaffPaymentStep(state).valid : true
 
-  const isEffectiveDateTimeValid = (isBaseCompany(state) && isStaff)
+  const isEffectiveDateTimeValid = isBaseCompany(state)
```

Comment:
Oh right, this WAS changed since FED was implemented.

Could you please update the corresponding unit test?
https://github.com/bcgov/business-create-ui/blob/49b35ce721bceb3f7f85a1be2f62a230b9324da9/tests/unit/ReviewConfirmDissolution.spec.ts#L61

Reply:
You bet, done and done!

Reply:
Original implementation was Staff only i believe, so this definitely made sense then!

Reply:
I was thinking you'd remove the staff code from the unit test function, since it's no longer relevant.

Reply:
Glad you fixed the function description though :)

Reply:
Ah i see, well it doesn't hurt either :) Not a bad thing to test for both roles right!

Reply:
We already know it renders from other tests, so i figured this was better than just removing the test.
But we can certainly do that if you prefer. I'm good with anything!

Reply:
No, it doesn't hurt, but it's an example of the undesirable "decoy pattern"... where code looks like it's doing something but it's a red herring.
(Enough metaphors in here for you? haha)

Reply:
Will remove! 😁